### PR TITLE
Upgrade page management services

### PR DESCRIPTION
**Pull request by eugeniosant**
Upgrade page management services and add Infinispan connector

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: plugins/entando-plugin-jpcollaboration/src/main/java/com/agiletec/plugins/jpcrowdsourcing/apsadmin/idea/IdeaAction.java [hunk 1]
```diff
@@ -1,237 +1,244 @@
...
+		} catch (Throwable t) {
+			_logger.error("Errore loading categories leafs", t);
+			throw new RuntimeException("Errore loading categories leafs");
+		}
+		return categories;
+	}
+
+	private void addSmallCategory(List<SmallCategory> categories, Category parentCat, String langCode, boolean completeTitle) {
+		String[] childrenCodes = parentCat.getChildrenCodes();
+		for (int i = 0; i < childrenCodes.length; i++) {
```
Comment (joewhite101) [inline plugins/entando-plugin-jpcollaboration/src/main/java/com/agiletec/plugins/jpcrowdsourcing/apsadmin/idea/IdeaAction.java]: Does this need a null check?

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
All is Well

**Pull request merged**